cli/codeFormat.js

```diff
@@ -0,0 +1,24 @@
+const prettier = require('prettier');
+const fs = require('fs');
+
+const walkSync = function (dir, filelist) {
```

Comment:
This is used by generated project so generate this file in the webpack/utils folder and use that